fileTemplates/Stateful Workflow.kt

```diff
@@ -1,35 +1,25 @@
-#set( $WorkflowName = "${NAME}Workflow" )
 package ${PACKAGE_NAME}
 
 import com.squareup.workflow1.Snapshot
 import com.squareup.workflow1.StatefulWorkflow
-import ${PACKAGE_NAME}.${WorkflowName}.Output
-import ${PACKAGE_NAME}.${WorkflowName}.Props
-import ${PACKAGE_NAME}.${WorkflowName}.Rendering
-import ${PACKAGE_NAME}.${WorkflowName}.State
 
 #parse("File Header.java")
-class ${WorkflowName} : StatefulWorkflow<Props, State, Output, Rendering>() {
-
-  data class Props
-  data class State
-  data class Output
-  data class Rendering
```

Comment:
This is the part where it gets trickier as suggesting this pattern of internal data classes is a goal.

Reply:
Not sure how complex you want it to get, but there could be an option to leave the template blank for at type and then these would be generated?

Reply:
cool. Didn't know about the internal pattern. I'll experiment with an if --> auto generate.

Reply:
ok @steve-the-edwards I think I have it working. If you leave an "optional" field blank then it will auto generate inner class (and import)